docs/execution_nodes.md

```diff
@@ -76,3 +76,28 @@ Wait a few minutes for the periodic AWX task to do a health check against the ne
 ## Removing instances
 
 You can remove an instance by clicking "Remove" in the Instances page, or by setting the instance `node_state` to "deprovisioning" via the API.
+
+## Troubleshooting
+
+### Fact cache not working
+
+Make sure the system timezone on the execution node matches the timezone on the control plane nodes.
```

Comment:
Is this true? Or does the comparison happen between AWX TZ and the execution nodes?